club/theme.json

```diff
@@ -287,8 +342,9 @@
                     "text": "var(--wp--preset--color--foreground)"
                 },
                 "typography": {
-                    "fontSize": "var(--wp--preset--font-size--small)",
-                    "textTransform": "uppercase"
+                    "fontSize": "var(--wp--preset--font-size--default-title)",
+                    "textTransform": "uppercase",
+                    "textDecoration": "none"
```

Comment:
It doesn't apply to date because of the specificity. Set it under `core/post-date -> elements -> link`.

Reply:
done!